.gitignore

```diff
@@ -11,4 +11,16 @@ __pycache__
 
 #IntelliJ
 .idea
-*.iml
\ No newline at end of file
+*.iml
+
+#Setup
+MANIFEST
+build/
```

Comment:
Not sure if I got your suggestion. Because a lot of times I need to test the `setup.py` or build the entire `.deb` locally. And both process will generate again these files and folders. I don't want we commit/push them by accident.